src/subcommand/map_main.cpp

```diff
@@ -83,6 +83,7 @@ void help_map(char** argv) {
          << "    -R, --read-group NAME         for --reads input, add this read group" << endl
          << "output:" << endl
          << "    -j, --output-json             output JSON rather than an alignment stream (helpful for debugging)" << endl
+         << "    -%, --gaf                     output alignments in GAF format" << endl
```

Comment:
Are we sure we want a `-%` option? Couldn't we leave just the long `--gaf` option, or replace this and `-j` with something tat takes a string to define the target format? Maybe rename `--surject-to` and make surjection implicit for the formats that need it?